src/js/protocols/stm32usbdfu.js

```diff
@@ -13,9 +13,9 @@
 'use strict';
 
 var STM32DFU_protocol = function () {
-    this.callback; // ref
-    this.hex; // ref
-    this.verify_hex;
+    this.callback = null;
+    this.hex = null;
+    this.verify_hex = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'verify_hex' is not in camel case.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5626377)